### PR TITLE
Vendor collections for isolated jobs to work in ansible 2.10

### PR DESCRIPTION
**Pull request by fosterseth**
##### SUMMARY

#6930 

install ansible modules necessary to run isolated jobs / container groups

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - API
 - Installer

##### AWX VERSION

```
awx: 11.2.0

```

### CODE REVIEW

File: requirements/collections_requirements.yml [hunk 1]
```diff
@@ -16,3 +16,7 @@ collections:
     version: 0.4.0  # first to contain necessary grouping and filtering features
   - name: ovirt.ovirt
     version: 1.0.0  # contains naming fix, was originally named ovirt.ovirt_collection
+  - name: community.kubernetes
+    version: 0.11.0
+  - name: ansible.posix
+    version: 0.1.0
```
Comment (AlanCoding) [inline requirements/collections_requirements.yml]: let's comment that these are required for the isolated management playbooks.

I think it makes sense to combine them right now - but they will be separated back out as execution environments start to be used for inventory imports.

Comment (fosterseth) [reply requirements/collections_requirements.yml]: left comments

**Review by AlanCoding — APPROVED** [review]
if it works, I would go for it

File: awx/main/isolated/manager.py [hunk 2]
```diff
@@ -74,6 +74,7 @@ def build_runner_params(self, hosts, verbosity=1):
         env['ANSIBLE_RETRY_FILES_ENABLED'] = 'False'
         env['ANSIBLE_HOST_KEY_CHECKING'] = str(settings.AWX_ISOLATED_HOST_KEY_CHECKING)
         env['ANSIBLE_LIBRARY'] = os.path.join(os.path.dirname(awx.__file__), 'plugins', 'isolated')
+        env['ANSIBLE_COLLECTIONS_PATHS'] = settings.INVENTORY_COLLECTIONS_ROOT
```
Comment (shanemcd) [inline awx/main/isolated/manager.py]: We should rename this variable now that it's being used for more than just inventory plugins.

Comment (AlanCoding) [reply awx/main/isolated/manager.py]: It's in the name of the folder too, which seems like a bigger problem than the setting name.

My thinking is these will be replaced before it really matters.

Comment (shanemcd) [reply awx/main/isolated/manager.py]: Renaming a few things isn't hard. We dont know if / when this vendoring solution is going away. Shouldn't we spend a few extra seconds to make things make sense?

Comment (ryanpetrello) [reply awx/main/isolated/manager.py]: Why not just make the variable `settings.AWX_ANSIBLE_COLLECTIONS_PATHS` so it's super clear what it represents?

Comment (fosterseth) [reply awx/main/isolated/manager.py]: going with Ryan's suggestion, thanks!

**Review by shanemcd — COMMENTED** [review]
LGTM, but please rebase / squash some commits before merging this.

**Review by AlanCoding — APPROVED** [review]
once the tests show that it's working, then go ahead and get it moving

thanks for doing it!

**Comment by fosterseth** [comment]
regate

**Comment by fosterseth** [comment]
recheck

**Comment by fosterseth** [comment]
recheck

**Comment by fosterseth** [comment]
recheck

**Pull request merged**